### ISSUE TITLE
Cannot connect to primary

### ISSUE THREAD

**Issue by taplar** [Studio3T/robomongo#1290]
Upgrading from 0.9.0 to 1.0.0.  Downloaded the stand alone zip and put it my directory as a sibling to the 0.9.0 directory.  Started it up and it had all my previous server configurations, which is cool.

I tried to create a new one based off of two existing definitions which were apart of a replica set.  Everything between the two definitions is the same except in the new one i made it a replica set and added the server:port for both of them in the replica set list.  It is unable to connect.

I then tried to clone one of the existing definitions and change it to be a replica set rather than a direct connection and it encounters the same error.

```
Cannot connect to replica set "thename" [lmdbdev3.mydomain.com:27017]
Set's primary is unreachable.

Reason:
No member of the set is reachable.
```

I tested the old connection with the L node and it works still and it is the primary.  It is odd that it will not connect and it is also odd that that error message popup does not list the other node of the set in the message.

**Comment by taplar** [comment]
It looks like if we go into the configuration file and manually set the replicaSet.setName it starts working.  However the option to set this does not exist in the gui.

**Comment by BillLeecn** [comment]
It seems that it requires a certain permission to get the replication set's name.
When I am authenticated as a clusterAdmin role, it's OK. However,  if I try to authenticate as a user without such permissions, this error occurs.

**Comment by simsekgokhan** [comment]
Hi All, sorry about this inconvenience, it should work with admin users. 
We have already done initial investigation and found the root cause of the problem. 

Please see the workaround solution and investigation details here:
https://github.com/paralect/robomongo/issues/1288#issuecomment-282002406

And please subscribe to this issue https://github.com/paralect/robomongo/issues/1288 for updates.

**Comment by simsekgokhan** [comment]
Hi, Replica set name configurable on UI feature is available on latest release Robomongo 1.0.

https://robomongo.org/download
http://blog.robomongo.org/robomongo-1-0/

I will close this ticket now, feel free to reopen.

**Comment by simsekgokhan** [comment]
Closing as duplicate of https://github.com/Studio3T/robomongo/issues/1288
